### DISCUSSION TITLE
Consiglio led strip zigbee

### DISCUSSION THREAD

**Discussion by H4Z4RD-H42** [indomus/forum#2286] [Hardware & Firmware]
Ciao raga, pensavo fosse più semplice districarsi tra i vari modelli di strisce led, ma mi sono reso conto che ce ne sono veramente un'infinità e non riesco a capire di cosa ho bisogno, oltre alla striscia, per realizzare questo semplice progetto.. Di sicuro deve essere ZigBee, RGBW, economica (diciamo sui 30€) e non waterproof.

La devo posizionare sotto l'isola della cucina, su entrambi i lati e ovviamente devono essere collegate tra di loro e non essere due entità separate. Ho visto che ci sono prolunghe, connessioni varie, sdoppiatori, raccordi... Mi aiutate gentilmente a capire cosa devo comprare su Amazon? Ecco uno stupidissimo schema:

![led_isola](https://user-images.githubusercontent.com/6650832/212736374-7752b368-a068-48ed-b05e-7d87162ed19e.jpg)

Le linee verdi sono le strisce led, la scatola rossa è la presa della corrente sul muro. Come si fa per collegare le 2 strisce insieme e poi andare alla presa della corrente senza che si vedano fili e alimentatori?

Può sembrare una sciocchezza lo so, ma stavo per acquistare il tutto per poi rendermi conto che se avessi preso quello che avevo visto io, mi sarei ritrovato con l'alimentatore da attaccare all'estremità di una delle due strisce e quindi dal lato opposto a dove sta la presa della corrente.

In pratica dovrei fare una cosa del genere:

![led_isola](https://user-images.githubusercontent.com/6650832/212737929-d184c4c1-d101-4d22-93a4-abe829c862b6.jpg)

Cioè collegare le 2 strisce con non so cosa (linea curva blu) e poi andare alla presa della corrente.

Purtroppo non posso collegare tra loro le due strisce dal lato sinistro dell'isola, perchè i lati corti dell'isola non hanno lo zoccolo e non saprei dove passare il filo senza che questo si veda.. devo per forza sfruttare il lato destro dell'isola perchè essendo quasi attaccato al muro, riesco a nascondere il tutto. SI può fare? Se si come?

Grazie!

**Comment by H4Z4RD-H42** [comment]
Chiedo scusa per il doppio post.. ho studiato un po', ho capito come collegare il tutto, mi serve semplicemente un raccordo a Y come questo
![UTa4nHE](https://user-images.githubusercontent.com/6650832/212977722-9fd81a6d-a701-45fc-896b-71797023e6e6.png)
e due strisce led da 2,40m l'una (una da 5m da tagliare a metà praticamente) da 12/24v e 60 led per metro. 

Non vorrei però commettere errori e chiedo sempre a voi un aiuto su quale controllore e quale alimentatore acquistare. Basta che sia zigbee e quindi integrabile su HA.

**EDIT:**
Ci ho preso? È tutto corretto? La striscia led c'è scritto che è da 60W totali.

![AOGJEB2 1](https://user-images.githubusercontent.com/6650832/212988183-1614b9e4-ee3c-43bc-8bfc-45a109476869.png)

**Reply by Scialla** [reply]
vuoi illuminazione standard o vuoi creare effetti di luce, animazioni, altro? Se si considera anche un ESP32 con firmware WLED e striscia 2816, anche se sarebbe wifi e non zigbee. Guarda su youtube le animazioni incredibili che si riescono a fare gestendo ogni led singolarmente

**Reply by H4Z4RD-H42** [reply]
Si le ho viste, parli delle indirizzabili vero? Probabilmente le utilizzerò in futuro ma per adesso mi serve solo una luce diciamo di cortesia/compagnia in modo che quando mi sveglio di notte per andare a bere, si accenda automaticamente e mi faccia quel minimo di luce per non prendere qualche muro in faccia e non accendere le luci normali.

Detto ciò, dici che i prodotti che ho scelto sono giusti e compatibili l'uno con l'altro?

Grazie mille.

**Reply by Scialla** [reply]
Non ho esperienze con quei prodotti, uso due strisce LIDL :)
Ti dico però che su una striscia con WLED da 5 mt, alimentandola con alimentatore simile a quello che hai indicato ma da 20A, da metà striscia in poi i led bianchi diventavano gialli. Ho dovuto alimentarla da entrambi i lati e, anche così, un paio di led a metà striscia tendono al giallo. Ho visto che tu li alimenterai solo da una parte; è possibile (magari non lo fa eh) che al fondo delle strisce il bianco tenda al giallo. Calcola bene poi se i 4,5A siano sufficienti. In ogni caso, se non fossero sufficienti, funzionerebbero lo stesso ma non raggiungeresti la massima luminosità possibile, quindi magari ti va bene lo stesso e chissene, al massimo il bianco tenderà un pò al giallino..

**Comment by H4Z4RD-H42** [comment]
Ammazza, 20A sono un botto e hai comunque problemi di perdita? Io ci capisco poco di corrente, ma leggendo in rete mi pare di aver capito che una volta appurato quanto consuma in Watt una striscia, si deve prendere orientativamente un alimentatore con una potenza del 20% superiore. Quindi se 5 metri di striscia sono 60W, un alimentatore da 100W dovrebbe essere più che ok. Poi boh...

**Reply by Scialla** [reply]
lo avevo preso da 20A perché pensavo di collegarci una seconda striscia a 5 mt, cosa che poi non ho fatto. La "perdita" non dipende dall'amperaggio iniziale ma dalla lunghezza della striscia, infatti alimentandola da entrambi i lati, ho risolto.
